Cargo.toml

```diff
@@ -41,11 +41,11 @@ serde_cbor = { version = "0.11.2"}
 uuid = { version = "1.3", features = ["v4", "serde"] }
 sys-info = "0.9.1"
 
-config = "~0.13.3"
+config = { default-features = false, features = ["yaml"], version = "~0.13.3" }
```

Comment:
Could not find other configs that aren't `yaml`. See https://github.com/mehcode/config-rs/blob/master/Cargo.toml#L18

Reply:
We do load/store some JSON structures throughout Qdrant, but I don't think they ever reach `Config`.

Thanks for tagging this.